.github/pull_request_assignment.yml

```diff
@@ -15,7 +15,6 @@
       - kazrael2119
       - v-xuto
```

Comment:
pl remove everyone else and leave just my alias

Reply:
Done